src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -325,6 +325,34 @@ import { Font } from 'astro:assets';
 <Font cssVariable="--font-roboto" preload />
 ```
 
+If not specified or set to `false`, no preload links will be emitted. If set to `true`, all possible links will be included.
+
+#### Granular preloads
+
+<p>
+<Since v="5.15.0" />
+</p>
+
+If you need more control over which font file is preloaded, you can instead provide an array of objects:
```

Comment:
```suggestion
You may not always want to preload every font link, as this can block loading other important resources, or may download fonts that are not needed for the current page. 

To selectively control which font files are preloaded, you can provide an array of objects describing any combination of font `weight`, `style`, or `subset` to preload.
```
Maybe explain the object a bit first?